desktop/src/main/java/bisq/desktop/components/paymentmethods/FasterPaymentsForm.java

```diff
@@ -44,6 +44,10 @@
 
     public static int addFormForBuyer(GridPane gridPane, int gridRow,
                                       PaymentAccountPayload paymentAccountPayload) {
+        if (!((FasterPaymentsAccountPayload) paymentAccountPayload).getHolderName().isEmpty()) {
+            addCompactTopLabelTextField(gridPane, ++gridRow, Res.get("payment.account.owner"),
+                    ((FasterPaymentsAccountPayload) paymentAccountPayload).getHolderName());
+        }
```

Comment:
If this field is shown during the trade process it creates unnecessary scrolling.
<img width="1312" alt="Bildschirmfoto 2020-03-26 um 16 16 33" src="https://user-images.githubusercontent.com/170962/77663881-d8603400-6f7d-11ea-918c-ed11a5557106.png">
I'd just change line 54-55 to
```
addCompactTopLabelTextField(gridPane, gridRow, 1, Res.get("payment.accountNr"),
                ((FasterPaymentsAccountPayload) paymentAccountPayload).getAccountNr());
```
which puts sort code and account nr next to each other.
<img width="1312" alt="Bildschirmfoto 2020-03-26 um 16 19 57" src="https://user-images.githubusercontent.com/170962/77663992-ff1e6a80-6f7d-11ea-8847-11f72f189964.png">
That would work for both cases.

Reply:
OK, I'll apply the suggested change.